release/community-stable/parrot/docker/Dockerfile

```diff
@@ -66,8 +67,15 @@ RUN \
     && dpkg -i /tmp/libicu57_57.1-6+deb9u3_amd64.deb \
     # remove libicu57 package
     && rm -f /tmp/libicu57_57.1-6+deb9u3_amd64.deb \
+    # upgrade distro
+    && apt-get dist-upgrade -y \
+    # clean downloaded packages
+    && apt-get clean \
+    # remove package list
+    && rm -rf /var/lib/apt/lists/*
+
     # install powershell package
-    && apt-get install -y /tmp/powershell.deb \
+RUN apt-get install -y /tmp/powershell.deb \
```

Comment:
I feel like this extra layer really isn't needed

Reply:
I added it during debugging for faster perf.  I will remove.